bower.json

```diff
@@ -14,6 +14,7 @@
     "dist/angular-leaflet-directive.js"
   ],
   "dependencies": {
+    "Leaflet.vector-markers": "https://github.com/hiasinho/Leaflet.vector-markers",
```

Comment:
@molehillrocker Perhaps ask if the maintainer would publish it at Bower and/or create version tags at Github? I often find asking works pretty well. ;-)